### ISSUE TITLE
Little error in guide

### ISSUE THREAD

**Issue by ulrikpl** [kubeflow/website#2545]
"Setting up a management cluster" step #2

`cd "${MGMT_DIR}"`

In order to work it has to be:

`cd "${MGMT_DIR}"/management` 

Since the previous step downloads ${MGMT_DIR}"/management:
`kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 "${MGMT_DIR}"`

**Comment by 8bitmp3** [comment]
Hi @ulrikpl Thanks for flagging this. Is the guide you're referring to this one: [Management cluster setup](https://www.kubeflow.org/docs/gke/deploy/management-setup/#setting-up-a-management-cluster)? (source: https://github.com/kubeflow/website/blob/master/content/en/docs/gke/deploy/management-setup.md)

/assign  @Bobgy joeliedtke rmgogogo
(team GCP)

**Comment by ulrikpl** [comment]
Yes that's the one. The same error repeats in the Deploy using kubectl and
kpt <https://www.kubeflow.org/docs/gke/deploy/deploy-cli/> section, under

"Fetch packages using kpt" :
    2) Change to the Kubeflow directory:
        cd "${KF_DIR}"

Since the previous step downloads to "${KF_DIR}"/kubeflow
kpt pkg get ***@***.***
"${KF_DIR}"

**Comment by Bobgy** [comment]
@ulrikpl thanks, it does need some improvement.

For clarification, when you specify `MGMT_DIR=~/xxxx/management`, (the last folder is exactly `management`).

then there won't be an extra `management` folder created in `${MGMT_DIR}/management`
